types/aws-lambda/trigger/lex-v2.d.ts

```diff
@@ -49,17 +54,6 @@ export type LexV2IntentState =
     | 'ReadyForFulfillment'
     | 'Waiting';
 
-export interface LexV2Slot {
-    shape?: string;
-    value: LexV2SlotValue;
-}
```

Comment:
Breaking change to remove this, again - seems like someone could easily be using this. Should probably at least change this to a type alias to a union of the new specific types?

Reply:
To make sure I understand, you're suggesting including `type LexV2Slot = LexV2ScalarSlotValue | LexV2ListSlotValue` in case users are already using the type `LexV2Slot`?

Reply:
Yep.

Reply:
Done